modules/olm-understanding-operator-catalog-images.adoc

```diff
@@ -40,7 +40,7 @@ The following catalogs are distributed by Red Hat:
 |Certified software that can be purchased from link:https://marketplace.redhat.com/[Red Hat Marketplace].
 
 |`community-operators`
-|`registry.redhat.io/redhat/community-operator-index:latest`
+|`registry.redhat.io/redhat/community-operator-index:{tag}`
```

Comment:
This will transform to the `v{product-version}` per branch. So `v4.7` for the `enterprise-4.7` branch, `v4.6` for the `enterprise-4.6` branch, etc.